Library/Formula/mcabber.rb

```diff
@@ -3,8 +3,8 @@
 class Mcabber < Formula
   desc "Console Jabber client"
   homepage 'http://mcabber.com/'
-  url 'http://mcabber.com/files/mcabber-0.10.3.tar.bz2'
-  sha1 '9254f520cb37e691fb55d4fc46df4440e4a17f14'
+  url 'http://lilotux.net/~mikael/mcabber/files/mcabber-1.0.0.tar.bz2'
+  sha1 '209182eb15aa9a1780e88492781c1f0b2282044f'
```

Comment:
Please use `sha256` here, Thanks! We'll be mass updating everything soon, and the less we have to mass update the better.